src/test/playwright/support/requests/ExerciseAPIRequests.ts

```diff
@@ -491,4 +507,23 @@ export class ExerciseAPIRequests {
         }
         return await this.page.request.put(url, { data: exercise });
     }
+
+    /**
+     * Creates a team for a team-based exercise.
+     *
+     * @param exerciseId - The ID of the exercise for which to create a team.
```

Comment:
Most likely just a matter of preferences - I think that it is an ID is already explained with the variable name 

```suggestion
     * @param exerciseId - for which to create a team.
```

Reply:
If it's just subjective, I'll leave it as it is for now, as it looks a bit odd to me even though I get the point. Also for consistency with other comments.